docs/markdown/onboarding-skill.md

```diff
@@ -16,7 +16,7 @@ The machine is defined [here](../../src/ts/cmd/onboarding-skill/src/services/onb
 
 From each state, invalid messages will be responded to (as long as they are parsable and contain a proper sender) with notUnderstood.
 
-## Developer's note
+## Developer's notes
```

Comment:
maybe "Developer notes" the - 's - looks strange to me.

Reply:
Developer's note, developer's notes, developers' note, developers' notes are virtually the same in meaning and none is grammatically wrong. Personal preferences can vary but rather than following personal preferences we should decide on a style guide to follow in future.